sdk/@launchdarkly/observability-node/rollup.config.mjs

```diff
@@ -36,12 +36,14 @@ const config = {
 			format: 'es',
 			sourcemap: true,
 			exports: 'auto',
+			inlineDynamicImports: true,
```

Comment:
The otel packages changed from using `require` for lazy loading to using dynamic imports. So we either need this, or we need chunking. Considering this isn't in a browser I think a single file is fine.

Reply:
Well, rollup just fails. Basically if you want to keep the dynamic import, then you need chunked output, so the output has to be a directory instead of individual files. Doing this tells it to just embed it instead of dynamically loading it from a chunk. Which makes sense for like the web where you could defer the cost from the initial load.

When using a require it would just bundle it anyway.